File: EpamTaxiAngular/Models/Order.cs
﻿using System;

namespace EpamTaxiAngular.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public string FullName { get; set; }
        public DateTime OrderDate { get; set; }
        public DateTime? CarDeliveryTime { get; set; }
        public string FromLocation { get; set; }
        public string ToLocation { get; set; }
        public decimal Cost { get; set; }
        public int Status { get; set; }

        public string UserId { get; set; }
        public virtual User User { get; set; }
    }
}

File: EpamTaxiAngular/ClientApp/src/app/order.ts
export class Order {
    constructor(
        public orderId?: number,
        public fullName?: string,
        public contactNumber?: string,
        public orderDate?: Date,
        public carDeliveryTime?: Date,
        public fromLocation?: string,
        public toLocation?: string,
        public cost?: number,
        public status?: number) { }
}